pkg/delegate_test.go

```diff
@@ -5,56 +5,132 @@ import (
 	"testing"
 	"time"
 
+	"github.com/pkg/errors"
 	"github.com/smartcontractkit/libocr/offchainreporting2/types"
+	ktypes "github.com/smartcontractkit/ocr2keepers/pkg/types/mocks"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/mock"
-
-	ktypes "github.com/smartcontractkit/ocr2keepers/pkg/types"
 )
 
 func TestStart(t *testing.T) {
-	t.Skip("service start throws nil pointer deference")
-	d, err := NewDelegate(DelegateConfig{
-		Logger: ktypes.NewMockLogger(t),
-		LocalConfig: types.LocalConfig{
-			BlockchainTimeout:                  1 * time.Second,
-			ContractConfigTrackerPollInterval:  15 * time.Second,
-			ContractTransmitterTransmitTimeout: 1 * time.Second,
-			DatabaseTimeout:                    100 * time.Millisecond,
-			ContractConfigConfirmations:        1,
-		},
+	t.Run("fails to create the delegate with an empty config", func(t *testing.T) {
+		logger := ktypes.NewMockLogger(t)
+		logger.On("Debug", mock.Anything, mock.Anything).Once()
+
+		_, err := NewDelegate(DelegateConfig{
+			Logger: logger,
+		})
+
+		assert.Equal(t, err.Error(), "bad local config while creating new oracle: blockchain timeout must be between 1s and 20s, but is currently 0s; contract config tracker poll interval must be between 15s and 2m0s, but is currently 0s; contract transmitter transmit timeout must be between 1s and 1m0s, but is currently 0s; database timeout must be between 100ms and 10s, but is currently 0s; contract config block-depth confirmation threshold must be between 1 and 100, but is currently 0: failed to create new OCR oracle")
```

Comment:
this will break if libocr changes its wording. but i guess it's good bc it forces us to take another look at the plugin.